### ISSUE TITLE
Convert resources page to TOC layout

### ISSUE THREAD

**Issue by emersonthis** [flattenthecurve/guide#308]
The Resources page has gotten large and has multiple headings. I suggest we convert this to use the same `toc` layout (on the What to Do page) so that it's more navigable. 

@mverzilli or @nditada Let me know if you endorse this.

**Comment by rousik** [comment]
I was wondering if a table that allows sorting/filtering using the existing attributes might be good fit for this as more and more content shows up.

**Comment by emersonthis** [comment]
I like that idea but I have no idea how to implement it in a way that translators can maintain with only markdown. Maybe with lots of properties in the top content? I’m hesitant to “raise the bar” in this way given that we’re already struggling to maintain the simpler existing content across languages.

**Comment by rousik** [comment]
Resources are not actually translated as far as I know.

**Comment by nditada** [comment]
correct, resources are not translated, the idea is that they can be posted in the relevant language
